x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/deprecation_tab_content.tsx

```diff
@@ -28,6 +34,63 @@ export interface CheckupTabProps extends UpgradeAssistantTabProps {
   checkupLabel: string;
 }
 
+export const filterDeps = (level: LevelFilterOption, search: string = '') => {
+  const conditions: Array<(dep: EnrichedDeprecationInfo) => boolean> = [];
+
+  if (level !== LevelFilterOption.all) {
+    conditions.push((dep: EnrichedDeprecationInfo) => dep.level === level);
+  }
+
+  if (search.length > 0) {
+    // Change everything to lower case for a case-insensitive comparison
+    conditions.push((dep) => {
+      try {
+        const searchReg = new RegExp(search.toLowerCase());
```

Comment:
Have you considered using case insensitive matching instead of using `toLowerCase` on both strings?

```
const searchReg = new RegExp(search, 'i');
return searchReg.test(dep.message);
```

Reply:
👍 Good point. This was logic brought over from the old implementation. I updated to use case insensitive matching.